### ISSUE TITLE
[Mac OS] skipUpdateCheck plist item don't work as described in docs

### ISSUE THREAD

**Issue by Kawohl** [owncloud/client#5431]
@michaelstingl 
just as a reminder we need to fix 
https://doc.owncloud.com/desktop/2.2/autoupdate.html#preventing-automatic-updates-in-mac-os-x-environments

**Comment by michaelstingl** [comment]
I did a quick check and instructions [in the docs](**url**) don't make sense.

`com.owncloud.desktopclient.plist` isn't at this location:
```
$ cat /Library/Preferences/com.owncloud.desktopclient.plist
cat: /Library/Preferences/com.owncloud.desktopclient.plist: No such file or directory
```

I found this file in the user Library, but here this change don't has any effect:

<img width="1025" alt="mac_os_10_12 2" src="https://cloud.githubusercontent.com/assets/214010/21815122/da52bf66-d75b-11e6-9cbb-e5c5f64b636d.png">

Only copying `/Applications/owncloud.app/Contents/Resources/deny_autoupdate_com.owncloud.desktopclient.plist` to `/Library/Preferences/com.owncloud.desktopclient.plist` did the trick:
```
$ sudo cp /Applications/owncloud.app/Contents/Resources/deny_autoupdate_com.owncloud.desktopclient.plist /Library/Preferences/com.owncloud.desktopclient.plist
```

@owncloud/desktop-developers Is this something that should work, or better remove this from the docs?

<img width="725" alt="the_automatic_updater_ _owncloud_client_manual_2_2_5_documentation" src="https://cloud.githubusercontent.com/assets/214010/21816249/c4a0c240-d75f-11e6-8524-7e2dade51feb.png">


### Expected behaviour
Update prevention on Mac OS should work as described in the docs.

### Actual behaviour
Update prevention on Mac OS don't work as described in the docs.

### Steps to reproduce
1. Follow the docs

### Server configuration

Server not needed.

### Client configuration
Client version:
2.2.3

Operating system:
Mac OS 10.12

OS language:
EN

Installation path of client:
/Applications/owncloud.app

**Comment by guruz** [comment]
I think your suggestion to just remove that striked out part from the docs is fine

**Comment by settermjd** [comment]
After a bit of research, I've found the file. It's not stored in the global preferences directory `/Library/Preferences/` but in the user's preferences directory `~/Library/Preferences/`. 

Also, the file itself is a binary file, not an XML file. So, by default, it cannot be edited. However, using plutil, the property list utility, which comes with OSX, you can convert it to XML, make the required change, and convert it back to binary afterwards. 

Given that, I'll update the existing docs to reflect this information.

**Comment by guruz** [comment]
you can edit it with XCode like a normal file, without converting.. but I this is really not worth the effort, copying is easier? :)

**Comment by settermjd** [comment]
Fair enough. It was an interesting exercise, if nothing else.

**Comment by settermjd** [comment]
Closing, as this is addressed by #5473
